packages/fast-layouts-react/src/horizontal-overflow/horizontal-overflow.utilities.ts

```diff
@@ -0,0 +1,56 @@
+import { HorizontalOverflowProps } from "./horizontal-overflow.props";
+import { ButtonDirection } from "./horizontal-overflow";
+
+export type EaseInOutQuad = (currentTime: number, startValue: number, changeInValue: number, duration: number) => number;
+export type AnimateScroll = () => void;
+
+export function getItems(props: HorizontalOverflowProps): React.ReactNode[] {
+    const children: React.ReactNode[] = Array.isArray(props.children)
+        ? props.children
+        : props.children
+        ? [props.children]
+        : [];
+
+    return children.filter((child: JSX.Element) => {
+        return child.props && child.props.slot !== ButtonDirection.previous && child.props.slot !== ButtonDirection.next;
+    });
+}
+
+/**
+ * Inspired by the github gist contribution: https://gist.github.com/andjosh/6764939
+ */
+export function scrollLeft(element: HTMLDivElement, left: number, duration: number): void {
```

Comment:
I still don't think this file should exist. If these are fully reusable outside the context of this component then lets figure out file structure that enables that. If they aren't, they should be private members on the class so we're not allowing exports of functions that were never written to be used outside a specific use-case and context.